.github/dependabot.yml

```diff
@@ -3,43 +3,84 @@ updates:
   - package-ecosystem: "gomod"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "monthly"
     labels:
       - "kind/dependencies"
+      - "area/engine"
+      - "area/cli"
+      - "gomod"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
+
   - package-ecosystem: "docker"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "monthly"
     labels:
       - "kind/dependencies"
+      - "tooling"
+      - "docker"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
+
+  - package-ecosystem: "gomod"
+    directory: "/internal/mage"
+    schedule:
+      interval: "monthly"
+    labels:
+      - "kind/dependencies"
+      - "tooling"
+      - "gomod"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
+
   - package-ecosystem: "npm"
     directory: "/sdk/nodejs"
     schedule:
-      interval: "daily"
+      interval: "monthly"
     labels:
       - "kind/dependencies"
       - "sdk/nodejs"
+      - "npm"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
 
   - package-ecosystem: "npm"
     directory: "/website"
     schedule:
-      interval: "daily"
+      interval: "monthly"
     labels:
       - "kind/dependencies"
-      - "area/website"
+      - "area/docs"
+      - "npm"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
 
   - package-ecosystem: "gomod"
     directory: "/sdk/go"
     schedule:
-      interval: "daily"
+      interval: "monthly"
     labels:
       - "kind/dependencies"
       - "sdk/go"
+      - "gomod"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
 
   - package-ecosystem: "pip"
     directory: "/sdk/python"
     schedule:
-      interval: "daily"
+      interval: "monthly"
     labels:
       - "kind/dependencies"
       - "sdk/python"
+      - "pip"
```

Comment:
Not sure if adding this label is worth it. Dependabot PRs include the `directory`, so this is how I filter python's dependencies:
https://github.com/dagger/dagger/pulls?q=is%3Apr+%22in+%2Fsdk%2Fpython%22+author%3Aapp%2Fdependabot+is%3Aopen

Reply:
Or use `label:sdk/python` but at one point these weren't being added.